confidant/routes/jwks.py

```diff
@@ -0,0 +1,112 @@
+import logging
+
+from flask import blueprints, jsonify
+
+from confidant import authnz
+from confidant.services.jwkmanager import jwk_manager
+from confidant.schema.jwks import jwt_response_schema, JWTResponse, \
+    jwks_response_schema, JWKSResponse
+
+
+logger = logging.getLogger(__name__)
+blueprint = blueprints.Blueprint('jwks', __name__)
+
+
+@blueprint.route('/v1/jwks/token', methods=['GET'])
+@authnz.require_auth
+def get_token():
+    """
+    Returns a JWT for the authenticated service
+
+    **Example request**:
+
+    .. sourcecode:: http
+
+       GET /v1/jwks/token
+
+    **Example response**:
+
+    .. sourcecode:: http
+
+       HTTP/1.1 200 OK
+       Content-Type: application/json
+
+       {
+         "token": "ey..."
+       }
+
+    :resheader Content-Type: application/json
+    :statuscode 200: Success
+    :statuscode 400: JWTs are not supported for this user
+    """
+    user = authnz.get_logged_in_user()
+    if authnz.user_is_service(user):
+        payload = {
+            'service': user,
+            'namespace': user.replace('-iad', '')
+        }
+
+        if 'development' in user or 'unauthenticated' in user:
+            token = jwk_manager.get_jwt('localdev', payload)
```

Comment:
need to remove this, as this is lyft specific.  we should pass this is in as parameter